src/sentry/receivers/project_detectors.py

```diff
@@ -44,6 +54,43 @@ def create_project_detectors(instance, created, **kwargs):
             sentry_sdk.capture_exception(e)
 
 
+def create_metric_detector_with_owner(project: Project, user=None, user_id=None, **kwargs):
+    """
+    Creates default metric detector when project is created, with the team as owner.
+    This listens to project_created signal which provides user information.
+    """
+
+    if not features.has("organizations:default-anomaly-detector", project.organization, actor=user):
+        return
+
+    owner_team = project.teams.first()
+
+    if owner_team is None:
+        logger.info(
+            "create_metric_detector_with_owner.no_team",
+            extra={"project_id": project.id, "organization_id": project.organization_id},
+        )
+
+    try:
+        enabled = features.has(
+            "organizations:anomaly-detection-alerts", project.organization, actor=user
+        )
+        detector = _ensure_metric_detector(
+            project, owner_team_id=owner_team.id if owner_team else None, enabled=enabled
+        )
```

Comment:
Is this so that an org on a plan w/o anomaly detection would have the default detector in the case that they do upgrade? FYI I have a task in my team's backlog to address an edge case I think you're likely to hit here - Seer won't keep data on detectors they're not receiving updates for over 90 days for, so when they upgrade the alert might not work. Here is the task: https://getsentry.atlassian.net/browse/ACI-528